week2/project/ecommerce/src/CategoryList.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
```

Comment:
```suggestion
import { useEffect, useState } from "react";
```

Nice-to-have:
When importing a React component, you can skip 'import React' for modern React (17+)
[Reference](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#:~:text=With%20the%20new%20transform%2C%20you%20can%20use%20JSX%20without%20importing%20React.)

Also can implement Eslint to avoid unused variables
<img width="634" alt="Screenshot 2025-02-09 at 02 40 02" src="https://github.com/user-attachments/assets/a9b7fe9d-d3ad-4879-aba3-b7ce393d5ff9" />